lib/src/cam_helpers.cc

```diff
@@ -48,11 +44,9 @@ std::string serialize(const cxxtools::SerializationInfo& si)
     std::string returnData("");
 
     try {
-        std::stringstream        output;
-        cxxtools::JsonSerializer serializer(output);
-        serializer.serialize(si);
+        cxxtools::SerializationInfo tmpCpy(si);
+        returnData = JSON::writeToString(tmpCpy, false);
```

Comment:
i think that local cpy is now out of context.
fty-common-json.h api was updated to handle const& si....